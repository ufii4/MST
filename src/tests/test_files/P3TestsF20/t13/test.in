n 4
edge_count
clear
i 1;2;3
i 0;1;4
i 0;2;5
i 0;3;6
i 1;3;5
i 2;3;7
mst
i 1;0;1
i 2;0;1
i 3;0;1
i 2;1;1
i 3;1;1
i 3;2;1
mst
edge_count
clear
exit

