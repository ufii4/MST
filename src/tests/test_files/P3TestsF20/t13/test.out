success
edge count is 0
success
success
success
success
success
success
success
mst 12.00
success
success
success
success
success
success
mst 3.00
edge count is 6
success
